src/introspection.c

```diff
@@ -137,7 +137,7 @@ _dispatch_introspection_thread_add(void)
 	_dispatch_unfair_lock_unlock(&_dispatch_introspection.threads_lock);
 }
 
-static void
+static __stdcall void
```

Comment:
is `__stdcall` defined by clang on darwin or linux?

Reply:
I get

```
<stdin>:1:6: warning: calling convention '__stdcall' ignored for this target
      [-Wignored-attributes]
void __stdcall test();
     ^
```

So yeah, it might be something you want to guard.

Reply:
Sure, I can control this by a macro